packages/@okta/vuepress-theme-prose/assets/css/layouts/_layout.scss

```diff
@@ -53,13 +53,18 @@ body {
   background-color: cv('brand');
 }
 
-.page-body {
+.page-body:not(.redesign) {
```

Comment:
I'm starting to wonder if we should set an `id` or `class` on the `body` element that we can globally scope our overrides on.